shared/constants/fs.js

```diff
@@ -484,8 +484,9 @@ export const viewTypeFromPath = (p: Types.Path): Types.FileViewType => {
 }
 
 export const generateFileURL = (path: Types.Path, address: string, token: string): string => {
-  const stripKeybase = Types.pathToString(path).slice('/keybase'.length)
-  return `http://${address}/files${stripKeybase}?token=${token}`
+  const stripKeybase = Types.pathToString(path).slice('/keybase/'.length)
```

Comment:
`'/keybase/'.length` should be a constant in this file.

Reply:
I actually find this easier to read, but don't think it bothers me much the other way. So will change.